src/OpenSsl.php

```diff
@@ -29,7 +31,6 @@
  */
 class OpenSsl
 {
-    use ErrorHandleTrait;
```

Comment:
So this is a `trait` which is required in this class to provide the `handleReturn` method. Please take a look at: http://php.net/manual/en/language.oop5.traits.php
We can not remove this line, also the use statment at the top is not needed since the class `ErrorHandleTrait` is already in the `PSX\OpenSsl` namespace